tests/test_profiles.py

```diff
@@ -240,3 +297,127 @@ def test_profile_conversion(self):
         zp = pp.to_fourierzernike(L=6, M=6, N=0)
         x = np.linspace(0, 1, 100)
         np.testing.assert_allclose(pp(x), zp(x), atol=1e-10)
+
+    @pytest.mark.unit
+    def test_default_profiles(self):
+        """Test that default profiles are just zeros."""
+        pp = PowerSeriesProfile()
+        sp = SplineProfile()
+        mp = MTanhProfile()
+        zp = FourierZernikeProfile()
+
+        x = np.linspace(0, 1, 10)
+        np.testing.assert_allclose(pp(x), 0)
+        np.testing.assert_allclose(sp(x), 0)
+        np.testing.assert_allclose(mp(x), 0)
+        np.testing.assert_allclose(zp(x), 0)
+
+    @pytest.mark.unit
+    def test_solve_with_combined(self):
+        """Make sure combined profiles work correctly for solving equilibrium.
+
+        Test for gh issue #347
+        """
+        ne = PowerSeriesProfile(3.0e19 * np.array([1, -1]), modes=[0, 10])
+        Te = PowerSeriesProfile(2.0e3 * np.array([1, -1]), modes=[0, 2])
+        Ti = Te
+        pressure = elementary_charge * (ne * Te + ne * Ti)
+        print("pressure params:", pressure.params)
+
+        LM_resolution = 6
+        eq1 = Equilibrium(
+            pressure=pressure,
+            iota=PowerSeriesProfile([1.61]),
+            Psi=np.pi,  # so B ~ 1 T
+            NFP=1,
+            L=LM_resolution,
+            M=LM_resolution,
+            N=0,
+            L_grid=2 * LM_resolution,
+            M_grid=2 * LM_resolution,
+            N_grid=0,
+            sym=True,
+        )
+        eq1.solve(
+            constraints=get_fixed_boundary_constraints(kinetic=False),
+            objective=ObjectiveFunction(objectives=ForceBalance()),
+            maxiter=5,
+        )
```

Comment:
FYI this should be equivalent to the simpler: `eq1.solve(objective-"force", maxiter=5)`. Same for `eq2`.

Reply:
Yes I just wanted it to be more explicit and clearer what the difference was (kinetic vs non)